### ISSUE TITLE
LSTM model save warning, Tensorflow 2.7.0

### ISSUE THREAD

**Issue by StevenHuang2020** [tensorflow/tensorflow#54090]
**System information**
- OS Platform and Distribution: Windows10
- TensorFlow installed from: binary
- TensorFlow version: v2.7.0-rc1-69-gc256c071bb2 2.7.0
- Python version: 3.9.10

**Describe the current behavior**
When model.save called, the below warning message occurred:
```
WARNING:absl:Found untraced functions such as lstm_cell_layer_call_fn, lstm_cell_layer_call_and_return_conditional_losses, lstm_cell_layer_call_fn, lstm_cell_layer_call_and_return_conditional_losses, lstm_cell_layer_call_and_return_conditional_losses while saving (showing 5 of 5). These functions will not be directly callable after loading.
WARNING:absl:<keras.layers.recurrent.LSTMCell object at 0x0000021419A6A820> has the same name 'LSTMCell' as a built-in Keras object. Consider renaming <class 'keras.layers.recurrent.LSTMCell'> to avoid naming conflicts when loading with `tf.keras.models.load_model`. If renaming is not possible, pass the object in the `custom_objects` parameter of the load function.
```
I just upgraded python from 3.6 to 3.9, reinstall Tensorflow under Python 3.9.10. I found the LSTM model hard to train to reduce loss value and the previous warning information occurred.

![image](https://user-images.githubusercontent.com/61686583/151123541-210c3046-b23a-4a09-812d-34bb2dde9ad3.png)

**Comment by StevenHuang2020** [comment]
My model creation code is as follows:
```
from tensorflow.keras.layers import Dense, LSTM
from tensorflow.keras.models import Sequential
from tensorflow.keras import optimizers

def createModel(look_back=1):
    model = Sequential()
    model.add(LSTM(100, return_sequences=True, input_shape=(1, look_back)))
    # model.add(LSTM(100, activation='relu', return_sequences=True))
    # model.add(LSTM(50, activation='relu', return_sequences=True))
    model.add(Dense(30, activation='relu'))
    model.add(Dense(20, activation='relu'))
    model.add(Dense(10, activation='relu'))
    model.add(Dense(1))

    # fixed learning rate
    # lr = 1e-2

    # learning rate schedule
    lr = optimizers.schedules.ExponentialDecay(initial_learning_rate=1e-5,
                                               decay_steps=10000,
                                               decay_rate=0.99)

    # opt = optimizers.SGD(learning_rate=lr)
    # opt = optimizers.SGD(learning_rate=lr, momentum=0.8, nesterov=False)
    # opt = optimizers.RMSprop(learning_rate=lr, rho=0.9, epsilon=1e-08)
    opt = optimizers.Adam(learning_rate=lr)
    # opt = optimizers.Adadelta(learning_rate=lr)
    # opt = optimizers.Adagrad(learning_rate=lr)
    # opt = optimizers.Adamax(learning_rate=lr)
    # opt = optimizers.Nadam(learning_rate=lr)
    # opt = optimizers.Ftrl(learning_rate=lr)

    model.compile(optimizer=opt, loss='mse')
    # model.summary()
    return model
```

**Comment by mohantym** [comment]
Hi @StevenHuang2020 ! 
Please post this issue on [keras-team/keras repo.](https://github.com/keras-team/keras/issues)
To know more see;
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999) . Thanks!

**Comment by StevenHuang2020** [comment]
okay, thanks!

**Comment by mohantym** [comment]
@StevenHuang2020 ! Could you close this issue here then as it will be tracked in Keras repository?

**Comment by StevenHuang2020** [comment]
ok
